ktor-http/common/test/io/ktor/tests/http/UrlTest.kt

```diff
@@ -121,6 +121,18 @@ class UrlTest {
         assertEquals("/response-headers?message=foo%25bar", url.fullPath)
     }
 
+    @Test
+    fun testEqualsInQueryValue() {
+        val urlString =
+            "https://akamai.bintray.com/22/225b067044aa56f36590ef56d41e256cd1d0887b176bfdeec123ecccc6057790?__gda__=exp=1604350711~hmac=417cbd5a97b4c499e2cf7e9eae5dfb9ad95b42cb3ff76c5fb0fae70e2a42db9c&..."
```

Comment:
can you simplify this test value?

Reply:
Yep, but I would like to keep it as it is in the origin issue